Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG BASE_IMAGE=node:22.7-bookworm-slim
+
+FROM $BASE_IMAGE
+
+# Combine RUN instructions and clean up after installations
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    debian-keyring \
+    debian-archive-keyring \
+    apt-transport-https && \
+    curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg && \
+    curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/debian.deb.txt' | tee /etc/apt/sources.list.d/caddy-stable.list && \
```

Comment:
both should be removd (see next comment)

Reply:
Will remove